composeApp/src/commonMain/kotlin/com/swmansion/routinetracker/App.kt

```diff
@@ -7,25 +7,34 @@ import androidx.compose.ui.Modifier
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
 import androidx.navigation.compose.rememberNavController
+import com.swmansion.routinetracker.data.createAlarmeePlatformConfiguration
 import com.swmansion.routinetracker.navigation.CreateRoutine
 import com.swmansion.routinetracker.navigation.CreateTask
 import com.swmansion.routinetracker.navigation.Home
 import com.swmansion.routinetracker.screen.CreateRoutineScreen
 import com.swmansion.routinetracker.screen.CreateTaskScreen
 import com.swmansion.routinetracker.screen.HomeScreen
+import com.tweener.alarmee.AlarmeeService
+import com.tweener.alarmee.rememberAlarmeeService
 
 @Composable
 fun App() {
     MaterialTheme {
         val navController = rememberNavController()
+        val alarmeeService: AlarmeeService =
+            rememberAlarmeeService(createAlarmeePlatformConfiguration())
```

Comment:
The `AlarmeeService` instance would optimally live in the dependency container. See ”Without Compose” [here](https://github.com/Tweener/alarmee?tab=readme-ov-file#4-initialize-alarmeeservice).